<launch>

 <!-- joy node -->
  <node respawn="true" pkg="joy"
        type="joy" name="gamepad" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
  </node>

 
  <node pkg="pioneer_tools" type="joystick_teleop" name="joystick_teleop" args="joy:=/joy velocity:=/cmd_vel">
<!-- Axes -->
  <param name="axis_linear" value="3" type="int"/>
  <param name="axis_angular" value="2" type="int"/>
  <param name="scale_linear" value="1" type="double"/>
  <param name="scale_angular" value="1" type="double"/>
  </node>
  <node pkg="rxtools" type="rxconsole" name="console"/>

</launch>

